benefits/eligibility/views.py

```diff
@@ -90,22 +90,22 @@ def confirm(request):
 
     # GET from an already verified user, no need to verify again
     if request.method == "GET" and session.eligible(request):
-        eligibility = session.eligibility(request)
-        return verified(request, [eligibility.name])
+        return verified(request)
 
     unverified_view = reverse(ROUTE_UNVERIFIED)
 
     agency = session.agency(request)
     flow = session.flow(request)
-    types_to_verify = agency.type_names_to_verify(flow)
+    type_to_verify = flow.name
```

Comment:
Yep, since all the analytics-related events are a subclass of `EligibilityEvent`, this [line](https://github.com/cal-itp/benefits/blob/25048d98b5a3842d783aeae91b98645324250c48/benefits/eligibility/analytics.py#L14) turned out to be a good place to put the conversion to a `list` instead of having it at different places in `views`.